lib/platform/bitbucket-server/index.ts

```diff
@@ -1,4 +1,4 @@
-import url from 'url';
+import url, { URLSearchParams } from 'url';
```

Comment:
Why this?

Reply:
I shouldn't have done this here. As this was not imported, I thought of adding that. Let me delete it for now.

https://github.com/renovatebot/renovate/blob/fc0b3d3026bd823c1ec81bc2c9c35a79aec1f6b6/lib/platform/bitbucket-server/index.ts#L360-L365

Reply:
I see, in older nodejs there was no global `URLSearchParams`